docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,140 @@
+   you should update the permissions. For example:
+
+   ```code
+   chmod o+rx teleport
+   ```
+   
+1. Restart the `teleport` service.
+
+## Missing logins for single sign-on users
+
+If you use an external identity provider to enable single sign-on for users, you should be sure
+to assign logins for those users.
+
+### Symptom
+
+Users who have access to Teleport through an authentication connector for an external identity
+provider don't see any of the logins they need to access remote resources.
+
+### Solution
+
+To fix this issue, you can assign logins to your single sign-on users or modify the traits in 
+the Teleport roles assigned to users through their group membership in the external identity provider.
+For more information about using traits in roles, see [Role Templates](../access-controls/guides/role-templates.mdx).
+
+## Listing available servers
+
+In some cases,  running `tsh ls` or `tctl nodes ls` commands might include servers that are offline.
+For example, a server that has stopped sending a heartbeat to the Teleport Proxy Service
+might continue to be listed as available in the output of `tsh ls` or `tctl nodes ls` commands for
+10 minutes or more.
+
+### Symptom
+
+An unresponsive server is listed as available in the output of `tsh ls` or `tctl nodes ls` commands.
+For example, you might run either of these commands and see output similar to the following for
+servers that have previously sent a heartbeat signal to the Teleport Proxy Service even if
+one of these servers subsequently went offline:
+
+```
+Node Name      Address        Labels                  
+-------------- -------------- ----------------------- 
+ip-172-3-1-242 127.0.0.1:3022 hostname=ip-172-3-1-242 
+ip-172-3-1-75  ⟵ Tunnel       hostname=ip-172-3-1-75  
+ip-172-3-2-177 ⟵ Tunnel       hostname=ip-172-3-2-177
+```
+
+### Solution
+
+To investigate whether a server that previously sent a heartbeat has become unresponsive, you can run the 
+`tsh ls` or `tctl nodes ls` command with the `--format json` command-line option to see additional 
+information, including an expiration time. For example:
+
+```json
+    "kind": "node",
+    "version": "v2",
+    "metadata": {
+      "name": "c78612d9-dab4-497f-a4d8-59ddb7edc6e9",
+      "labels": {
+        "teleport.internal/resource-id": "3547a530-3b58-4f65-8335-c5cf99c7b374"
+      },
+      "expires": "2023-09-15T21:40:17.653190645Z",
+      "id": 1694813417653574518
+    },
+```
+
+If the server is sending a regular heartbeat signal, the `expires` value should be
+approximately eight to ten minutes from the current time. If the time to expire is
+less eight to ten minutes from the current time—for example, in the next two minutes—it's
+likely that the server has stopped sending the heartbeat.
+
+## Unable to join a shared session
+
+Moderated sessions or remote shared sessions enable multiple users to observe, participate, or terminate
```

Comment:
This sentence is not incorrect, but since it's not specific to moderated sessions I would generalize it a bit.

```suggestion
Teleport allows multiple users to observe or participate in
```